backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/TargetLawServiceTest.java

```diff
@@ -62,4 +65,28 @@ void canLoadTargetLawXmlByEli() {
     verify(loadTargetLawXmlAdapter, times(1))
         .loadTargetLawXmlByEli(argThat(command -> command.eli().equals(eli)));
   }
+
+  @Test
+  void appliesTimeMachineWhenCalled() {
+    // Given
+    final String targetLawEli = "someEli";
+    final String amendingLawXmlAsString = "</amendingLawXml>";
+
+    final TimeMachineUseCase.Query query =
+        new TimeMachineUseCase.Query(targetLawEli, amendingLawXmlAsString);
+
+    final String targetLawXmlAsString = "<targetLaw>Test content</targetLaw>";
+    when(loadTargetLawXmlAdapter.loadTargetLawXmlByEli(any()))
+        .thenReturn(Optional.of(targetLawXmlAsString));
+    when(timeMachine.apply(any(), any())).thenReturn("Success");
+
+    // When
+    final String appliedXml = service.applyTimeMachine(query);
+
+    // Then
+    assertThat(appliedXml).contains("Success");
+    verify(loadTargetLawXmlAdapter, times(1))
```

Comment:
I notice three asserts/verifies. Maybe the name of the test can be more specific (as to what this test makes sure of).